plugins/hypervisors/kvm/src/com/cloud/hypervisor/kvm/resource/BridgeVifDriver.java

```diff
@@ -287,7 +287,7 @@ private void deleteExitingLinkLocalRouteTable(String linkLocalBr) {
             for (String line : lines) {
                 String[] tokens = line.split(" ");
                 if (!tokens[2].equalsIgnoreCase(linkLocalBr)) {
```

Comment:
For this PR we cannot make any changes now that it is merged already; perhaps send a new PR or I can pick it up later.